crates/nixci/src/command/run.rs

```diff
@@ -9,13 +9,21 @@ use nix_rs::{
     flake::{system::System, url::FlakeUrl},
     info::NixInfo,
 };
+use std::path::PathBuf;
 
 use crate::{
     config::core::Config,
     flake_ref::FlakeRef,
-    nix::system_list::{SystemsList, SystemsListFlakeRef},
+    nix::{
+        ssh,
+        system_list::{SystemsList, SystemsListFlakeRef},
+    },
 };
 
+/// We expect this environment to be set in Nix build and shell.
```

Comment:
Generally, always document the "what" (rather than "why" or any other thing).

```suggestion
/// Path to Rust source corresponding to this (running) instance of Omnix
```